.github/actions/test-ruby/action.yml

```diff
@@ -40,6 +40,15 @@ runs:
         restore-keys: |
           ${{ runner.os }}-gems-
 
+    - name: Linting
+      run: |
+        bin/rails zeitwerk:check
+        bundle exec brakeman --skip-files bundle/ -q -w2
+        gem install rubocop
```

Comment:
I need to check with Bryan and Geoff why we were installing each time and not adding to the Gemfile. Perhaps we can assume it's installed here.

Reply:
that'd be great to have, the caching will do the job for us

Reply:
or could have a flags for each of the checks?

Reply:
we can, for sure. Just FYI flags will be enabled by default.

Reply:
changes were implemented